server/chat-commands/core.ts

```diff
@@ -320,7 +319,11 @@ export const commands: Chat.ChatCommands = {
 		}
 		return this.parse(`/msg ${user.lastPM || ''}, ${target}`);
 	},
-	replyhelp: [`/reply OR /r [message] - Send a message to the last user you got a message from, or sent a message to.`],
+	replyhelp: [
```

Comment:
Server-side ``/reply`` should probably be deprecated. Typing ``/r`` in the client should auto-replace into ``/dm USERNAME``, in order to avoid replying to the wrong user. Do you agree, @Zarel ?

Reply:
In theory it already does. I guess I don't remember if that's been ported to the new client. But yeah, server-side reply deprecated, fine by me.

Reply:
> In theory it already does. I guess I don't remember if that's been ported to the new client.

I'm not seeing that in any client. To clarify, I mean that the replacement should be performed before sending the message.